content/blog/2020/11/2020-11-03-gsoc-report.adoc

```diff
@@ -0,0 +1,163 @@
+This year these guidelines helped a lot because all the expectations were known in advance.
+As usual, there is a lot to improve, especially with regards to the community bonding phases and cross-project communications.
+We are processing the feedback, and we will expand our documentation and the contributor onboarding plans next year.
+
+== Some personal notes
+
+I have been involved in leading and coordinating Google Summer of Code in open source projects since 2016.
+This year I visited the GSoC stand at FOSDEM and met a few organizers and former students.
+A few days after, I proposed participating in GSoC at the Jenkins contributor summit in Brussels,
+and several contributors supported this idea.
+we spent several hours to create the first Jenkins GSoC page and brainstorm on project ideas.
+We submitted our application, and we got accepted.
+Thanks a lot to the Google team that gave us a chance!
+
+It is great to work with the students and see how they explore the open source community and grow professionally as engineers.
+It is also awesome to see how some of them stay in the project and keep contributing,
+including becoming plugin maintainers and GSoC mentors.
+But, for me, Google Summer of Code is not just about mentoring.
+It also helps a lot with the community bonding... for the existing community like Jenkins which has a lot of isolated sub-communities in plugins.
+Many maintainers work alone, and it is not the best place.
+When plugin contributors become project mentors, they join the wider community effort and work in teams.
+In many cases they start contributing to the organization-wide activities and goals,
+and it grows the "backbone" of the Jenkins community.
+Like other community-driven projects, we need such backbone to scale the community and onboard more contributors to the countless Jenkins components.
+So far it works really well and GSoC excels among outreach programs in this regard.
+
+// TODO:Move more notes from the draft
+
+I would like to thank the Google Open Source team, students and all Jenkins community members for the great Google Summer of Code this year.
+We also thank Continuous Deliver
+Last but not least, I would like to thank the Jenkins org admin team:
+link:/blog/authors/martinda[Martin d'Anjou],
+link:/blog/authors/markyjackson-taulia[Marky Jackson], and
+link:/blog/authors/marckk/[Kara de la Marck].
+This was a crazy year for everyone. Regardless of that, the org admins stepped up and took responsibility for students and mentors involved in the project, with a serious time commitment.
+Not all work by the organizers is publicly visible (applications, project selection, resolving conflicts),
+but this work is essential to the project's success.
+Thanks a lot to org admins and mentors who helped with the administrative tasks this year!
+
+== So, what about GSoC 2021?
+
+Yes, we are interested to participate in Google Summer of Code 2021.
```

Comment:
Not sure if "want to participate" or "plan to participate" express your intent.  (optional)
```suggestion
Yes, we plan to participate in Google Summer of Code 2021.
```